### ISSUE TITLE
xinference中内置的语音模型好像都或多或少有些问题,比如cosyvoice和SenseVoiceSmall不支持numpy2 ,whisper下载的时候都会失败,FishSpeech-1.4也是会下载失败

### ISSUE THREAD

**Issue by zhaoshuyan** [xorbitsai/inference#2417]
### System Info / 系統信息

centos7 absl-py                   2.1.0
accelerate                1.0.0
aiofiles                  23.2.1
aiohappyeyeballs          2.4.3
aiohttp                   3.10.9
aioprometheus             23.12.0
aiosignal                 1.3.1
alembic                   1.13.3
aliyun-python-sdk-core    2.16.0
aliyun-python-sdk-kms     2.16.5
altair                    5.4.1
annotated-types           0.7.0
antlr4-python3-runtime    4.9.3
anyio                     4.6.0
argon2-cffi               23.1.0
argon2-cffi-bindings      21.2.0
arrow                     1.3.0
asttokens                 2.4.1
async-lru                 2.0.4
async-timeout             4.0.3
attrs                     24.2.0
audioread                 3.0.1
autopage                  0.5.2
babel                     2.16.0
bcrypt                    4.2.0
beautifulsoup4            4.12.3
bibtexparser              2.0.0b7
bleach                    6.1.0
certifi                   2024.8.30
cffi                      1.17.1
cfgv                      3.4.0
charset-normalizer        3.4.0
click                     8.1.7
cliff                     4.7.0
clldutils                 3.22.2
cloudpickle               3.0.0
cmaes                     0.11.1
cmd2                      2.4.3
colorama                  0.4.6
coloredlogs               15.0.1
colorlog                  6.8.2
comm                      0.2.2
conformer                 0.3.2
contourpy                 1.3.0
crcmod                    1.7
cryptography              43.0.1
csvw                      3.3.1
cycler                    0.12.1
Cython                    3.0.11
debugpy                   1.8.6
decorator                 5.1.1
defusedxml                0.7.1
diffusers                 0.30.3
distlib                   0.3.9
distro                    1.9.0
dlinfo                    1.2.1
ecdsa                     0.19.0
editdistance              0.8.1
einops                    0.8.0
exceptiongroup            1.2.2
executing                 2.1.0
fastapi                   0.110.3
fastjsonschema            2.20.0
ffmpy                     0.4.0
filelock                  3.16.1
flatbuffers               24.3.25
fonttools                 4.54.1
fqdn                      1.5.1
frozenlist                1.4.1
fsspec                    2024.9.0
funasr                    1.1.11
gdown                     5.2.0
gradio                    3.43.2
gradio_client             0.5.0
greenlet                  3.1.1
grpcio                    1.66.2
h11                       0.14.0
httpcore                  1.0.6
httpx                     0.27.2
huggingface-hub           0.25.2
humanfriendly             10.0
hydra-colorlog            1.2.0
hydra-core                1.3.2
hydra-optuna-sweeper      1.2.0
HyperPyYAML               1.2.2
identify                  2.6.1
idna                      3.10
importlib_metadata        8.5.0
importlib_resources       6.4.5
inflect                   7.4.0
iniconfig                 2.0.0
ipykernel                 6.29.5
ipython                   8.28.0
ipywidgets                8.1.5
isodate                   0.6.1
isoduration               20.11.0
jaconv                    0.4.0
jamo                      0.4.1
jedi                      0.19.1
jieba                     0.42.1
Jinja2                    3.1.4
jiter                     0.6.1
jmespath                  0.10.0
joblib                    1.4.2
json5                     0.9.25
jsonpointer               3.0.0
jsonschema                4.23.0
jsonschema-specifications 2024.10.1
jupyter_client            8.6.3
jupyter_core              5.7.2
jupyter-events            0.10.0
jupyter-lsp               2.2.5
jupyter_server            2.14.2
jupyter_server_terminals  0.5.3
jupyterlab                4.2.5
jupyterlab_pygments       0.3.0
jupyterlab_server         2.27.3
jupyterlab_widgets        3.0.13
kaldiio                   2.18.0
kiwisolver                1.4.7
language-tags             1.2.0
lazy_loader               0.4
librosa                   0.10.2.post1
lightning                 2.4.0
lightning-utilities       0.11.7
llvmlite                  0.43.0
lxml                      5.3.0
Mako                      1.3.5
Markdown                  3.7
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matcha-tts                0.0.7.0
matplotlib                3.9.2
matplotlib-inline         0.1.7
mdurl                     0.1.2
mistune                   3.0.2
modelscope                1.18.1
more-itertools            10.5.0
mpmath                    1.3.0
msgpack                   1.1.0
multidict                 6.1.0
narwhals                  1.9.2
nbclient                  0.10.0
nbconvert                 7.16.4
nbformat                  5.10.4
nest-asyncio              1.6.0
networkx                  3.3
nodeenv                   1.9.1
notebook                  7.2.2
notebook_shim             0.2.4
numba                     0.60.0
numpy                     2.0.0
nvidia-cublas-cu12        12.1.3.1
nvidia-cuda-cupti-cu12    12.1.105
nvidia-cuda-nvrtc-cu12    12.1.105
nvidia-cuda-runtime-cu12  12.1.105
nvidia-cudnn-cu12         9.1.0.70
nvidia-cufft-cu12         11.0.2.54
nvidia-curand-cu12        10.3.2.106
nvidia-cusolver-cu12      11.4.5.107
nvidia-cusparse-cu12      12.1.0.106
nvidia-ml-py              12.560.30
nvidia-nccl-cu12          2.20.5
nvidia-nvjitlink-cu12     12.6.77
nvidia-nvtx-cu12          12.1.105
omegaconf                 2.3.0
onnxruntime-gpu           1.16.3
openai                    1.51.2
optuna                    2.10.1
orjson                    3.10.7
oss2                      2.19.0
overrides                 7.7.0
packaging                 24.1
pandas                    2.2.3
pandocfilters             1.5.1
parso                     0.8.4
passlib                   1.7.4
pbr                       6.1.0
peft                      0.13.1
pexpect                   4.9.0
phonemizer                3.3.0
pillow                    10.4.0
pip                       24.2
platformdirs              4.3.6
pluggy                    1.5.0
pooch                     1.8.2
pre_commit                4.0.1
prettytable               3.11.0
prometheus_client         0.21.0
prompt_toolkit            3.0.48
propcache                 0.2.0
protobuf                  5.28.2
psutil                    6.0.0
ptyprocess                0.7.0
pure_eval                 0.2.3
pyarrow                   17.0.0
pyasn1                    0.6.1
pycparser                 2.22
pycryptodome              3.21.0
pydantic                  2.9.2
pydantic_core             2.23.4
pydub                     0.25.1
Pygments                  2.18.0
pylatexenc                2.10
pynini                    2.1.6.post1
pynndescent               0.5.13
pyparsing                 3.1.4
pyperclip                 1.9.0
PySocks                   1.7.1
pytest                    8.3.3
python-dateutil           2.9.0.post0
python-dotenv             1.0.1
python-jose               3.3.0
python-json-logger        2.0.7
python-multipart          0.0.12
pytorch-lightning         2.4.0
pytorch-wpe               0.0.1
pytz                      2024.2
PyYAML                    6.0.2
pyzmq                     26.2.0
quantile-python           1.1
rdflib                    7.0.0
referencing               0.35.1
regex                     2024.9.11
requests                  2.32.3
rfc3339-validator         0.1.4
rfc3986                   1.5.0
rfc3986-validator         0.1.1
rich                      13.9.2
rootutils                 1.0.7
rpds-py                   0.20.0
rsa                       4.9
ruamel.yaml               0.18.6
ruamel.yaml.clib          0.2.8
safetensors               0.4.5
scikit-learn              1.5.2
scipy                     1.14.1
seaborn                   0.13.2
segments                  2.2.1
semantic-version          2.10.0
Send2Trash                1.8.3
sentencepiece             0.2.0
setuptools                75.1.0
six                       1.16.0
sniffio                   1.3.1
soundfile                 0.12.1
soupsieve                 2.6
soxr                      0.5.0.post1
SQLAlchemy                2.0.35
sse-starlette             2.1.3
stack-data                0.6.3
starlette                 0.37.2
stevedore                 5.3.0
sympy                     1.13.3
tabulate                  0.9.0
tblib                     3.0.0
tensorboard               2.18.0
tensorboard-data-server   0.7.2
tensorboardX              2.6.2.2
terminado                 0.18.1
threadpoolctl             3.5.0
tiktoken                  0.8.0
timm                      1.0.9
tinycss2                  1.3.0
tokenizers                0.20.0
tomli                     2.0.2
torch                     2.4.1
torch-complex             0.4.4
torchaudio                2.4.1
torchmetrics              1.4.2
torchvision               0.19.1
tornado                   6.4.1
tqdm                      4.66.5
traitlets                 5.14.3
transformers              4.45.2
triton                    3.0.0
typeguard                 4.3.0
types-python-dateutil     2.9.0.20241003
typing_extensions         4.12.2
tzdata                    2024.2
umap-learn                0.5.6
Unidecode                 1.3.8
uri-template              1.3.0
uritemplate               4.1.1
urllib3                   2.2.3
uvicorn                   0.31.1
uvloop                    0.20.0
virtualenv                20.26.6
wcwidth                   0.2.13
webcolors                 24.8.0
webencodings              0.5.1
websocket-client          1.8.0
websockets                11.0.3
Werkzeug                  3.0.4
WeTextProcessing          1.0.4.1
wget                      3.2
wheel                     0.44.0
whisper                   1.1.10
widgetsnbextension        4.0.13
xinference                0.15.3+0.g00a9ee1.dirty /home/zsy/xinference/inference-0.15.3
xoscar                    0.3.3
yarl                      1.14.0
zipp                      3.20.2

### Running Xinference with Docker? / 是否使用 Docker 运行 Xinfernece？

- [ ] docker / docker
- [ ] pip install / 通过 pip install 安装
- [X] installation from source / 从源码安装

### Version info / 版本信息

0.15.3

### The command used to start Xinference / 用以启动 xinference 的命令

XINFERENCE_HOME=/tmp/xinference xinference-local --host 0.0.0.0 --port 9997

### Reproduction / 复现过程

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.0.0 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2

### Expected behavior / 期待表现

期待内置的这些语音模型可以丝滑的下载使用

**Comment by qinxuye** [comment]
numpy2 目前还太新了，大家适配还没跟上。

whisper 和 fish speech 等目前有不少模型只在 hf 上有，所以可能会下载失败。

可以启动的时候添加 HF_ENDPOINT=https://hf-mirror.com

```
HF_ENDPOINT=https://hf-mirror.com xinference-local xxx
```
